lightning/src/util/sweep.rs

```diff
@@ -382,7 +382,8 @@ where
 		output_spender: O, change_destination_source: D, kv_store: K, logger: L,
 	) -> Self {
 		let outputs = Vec::new();
-		let sweeper_state = Mutex::new(SweeperState { outputs, best_block });
+		let sweeper_state =
+			Mutex::new(SweeperState { persistent: PersistentSweeperState { outputs, best_block } });
```

Comment:
If we're gonna add the wrapper struct, should expand the commit message here for why

Reply:
Added that we want to avoid the _unused construct.